relay-sampling/src/config.rs

```diff
@@ -162,6 +162,17 @@ pub enum SamplingValue {
         /// The limit of how many times this rule will be sampled before this rule is invalid.
         limit: i64,
     },
+
+    /// A minimum sample rate.
+    ///
+    /// The sample rate specified in the rule will be used as a minimum over the otherwise used
+    /// sample rate.
+    ///
+    /// Only the first matching minimum sample rate will be applied.
+    MinimumSampleRate {
+        /// The minimum sample rate used to raise the chosen sample rate.
+        value: f64,
+    },
```

Comment:
We could also express this as part of the `SampleRate` rule:

```rs
SampleRate {
    value: f64,
    tbd: max | min | constant
}
```

Reply:
I prefer the current version, mostly because I can't think of a good term for what you call `constant` above.